src/util/constants.ts

```diff
@@ -0,0 +1,5 @@
+// Farms
+export let WPOKT_GENESIS_FARM = '0xdb7d0a3bfcb7b34e6be510b26c55c8b60bd66d4a'
+
+// Subgraph
```

Comment:
Can we remove this and the above comment? I think they're not adding anything useful. What do you think?